src/groups/mqb/mqbblp/mqbblp_recoverymanager.cpp

```diff
@@ -1722,14 +1722,15 @@ int RecoveryManager::replayPartition(
         bmqp::StorageMessageType::Enum storageMsgType =
             bmqp::StorageMessageType::e_UNDEFINED;
 
-        rc = mqbc::RecoveryUtil::incrementCurrentSeqNum(&currentSeqNum,
-                                                        &journalRecordBase,
-                                                        fti->journalFd(),
-                                                        toSequenceNum,
-                                                        pid,
-                                                        *destination,
-                                                        *d_clusterData_p,
-                                                        journalIt);
+        rc = mqbc::RecoveryUtil::incrementCurrentSeqNum(
+            &currentSeqNum,
+            &journalRecordBase,
+            fti->journalFd(),
+            toSequenceNum,
+            pid,
+            *destination,
+            d_clusterData_p->identity().description(),
```

Comment:
Just making sure I haven't missed anything in this commit---do any changes here add thread-safety, or are they only additional checks/documentation + changing the clusterData pointer to a const ref?